library/data/ir_ui_menu.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+  <record id="point_of_sale.menu_point_root" model="ir.ui.menu" forcecreate="0">
+    <field name="sequence">1</field>
+  </record>
+  <record id="return_main_menu" model="ir.ui.menu">
+    <field name="name">Return</field>
+    <field name="sequence">2</field>
+    <field name="web_icon_data" type="base64" file="library/static/description/icon.png"/>
```

Comment:
I understand that here the icon should be without border, but it shouldn't be the case elsewhere. So what I suggest is to have both (with and without borders) and
- icon.png would be with borders -> necessary for appstore
- icon_without_border.png without
and here you'd use the icon_without_border.png
```suggestion
    <field name="web_icon_data" type="base64" file="library/static/description/icon_without_border.png"/>
```